azure-pipelines.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       imageName: 'libgit2/trusty-openssl:latest'
       environmentVariables: |
        CC=gcc
+       CMAKE_OPTIONS=-DUSE_HTTPS=OpenSSL -DSHA1_BACKEND=OpenSSL
```

Comment:
Please don't set `SHA1_BACKEND`.  We should encourage people to always use SHA1DC.

In fact, recent conversations on the git mailing list suggest that we should be talking about Git as using SHA1DC instead of using SHA1.  I haven't disabled the ability to choose a SHA1 backend anyway, because some consumers have various requirements about the cryptosystems that they use and I don't want to break them without better knowing their needs here.

Reply:
True, I didn't take this into account. I was just trying to make the testsuite fully specified, and SHA1DC is our default. Or should I drop that commit from the patch set ?

Reply:
I'm fine with the -DUSE_HTTPS=OpenSSL part, I think it makes sense to be explicit there.